azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -258,33 +245,16 @@ public void run() {
     }
 
     private void checkAllTriggers() throws TriggerManagerException {
-      long now = System.currentTimeMillis();
-
       // sweep through the rest of them
       for (Trigger t : triggers) {
         try {
           scannerStage = "Checking for trigger " + t.getTriggerId();
```

Comment:
shouldSkip here is never used after setting by this piece code. So I just removed it.